bags_api/src/main/scala/uk/ac/wellcome/platform/storage/bags/api/models/DisplayBagVersionList.scala

```diff
@@ -0,0 +1,50 @@
+package uk.ac.wellcome.platform.storage.bags.api.models
+
+import java.net.URL
+
+import io.circe.generic.extras.JsonKey
+import uk.ac.wellcome.platform.archive.bag_tracker.models.{BagVersionEntry, BagVersionList}
+import uk.ac.wellcome.platform.archive.common.bagit.models.BagId
+import uk.ac.wellcome.platform.archive.common.storage.models.StorageManifest
+
+case class DisplayBagVersionEntry(
```

Comment:
Where are these used?

Reply:
They're used to display a list of version, e.g. if you query

```
GET /bags/digitised/b1234/versions
```

They're renamed to match the models from the internal bag tracker classes.